src/components/overlay/index.tsx

```diff
@@ -105,24 +104,25 @@ class Overlay extends PureComponent<OverlayTypes> {
   };
 
   render() {
-    const {type, intensity, customContent} = this.props;
+    const {type, intensity, customContent, borderRadius} = this.props;
     const imageSource = this.getImageSource(type, intensity);
-
-    if (type === OVERLY_TYPES.VERTICAL) {
-      return (
-        <>
-          {this.renderImage([this.getStyleByType(OVERLY_TYPES.TOP), styles.vertical], imageSource)}
-          {this.renderImage([this.getStyleByType(OVERLY_TYPES.BOTTOM), styles.vertical], imageSource)}
-          {customContent && this.renderCustomContent()}
-        </>
-      );
-    }
+    const isVertical = type === OVERLY_TYPES.VERTICAL;
 
     return (
-      <>
-        {type && this.renderImage(this.getStyleByType(), imageSource)}
-        {customContent && this.renderCustomContent()}
-      </>
+      <View style={{overflow: 'hidden', borderRadius, ...StyleSheet.absoluteFillObject}}>
```

Comment:
This style can move to the styleSheet while borderRadiuse can be added to an array

Reply:
No need for absolute, pass flex